web/assets/javascripts/metrics.js

```diff
@@ -206,7 +188,7 @@ class HistBubbleChart extends BaseChart {
     });
 
     // Chart.js will not calculate the bubble size. We have to do that.
-    const maxRadius = this.el.offsetWidth / this.options.labels.length;
+    const maxRadius = this.el.offsetWidth / 100;
```

Comment:
This is an arbitrary number that results is reasonable looking bubbles. The previous implementation was equally arbitrary, but it relied on `labels`, which we're no longer passing.